demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
+        value_type: 1 # STRING
+      - name: target.name
+        value_type: 1 # STRING
+      - name: api.name
```

Comment:
We want the service the request is being routed to here: what is the proxy using to generate `source.service` (source.service is not defined in our attribute vocab, should it actually be api.name)?